README.md

```diff
@@ -1,13 +1,238 @@
+It also allows users to send/receive kin to/from different ecosystem applications.
+
+
+## Installation
+To include the library in your project add jitpack to your *build.gradle* project file.
+
+```gradle
+allprojects {
+    repositories {
+        ...
+        maven { url 'https://jitpack.io' }
+    }
+}
+```
+
+Add the latest release appsDiscovery version to your *build.gradle* file
+
+
+```gradle
+dependencies {
+    ...
+    implementation 'com.github.kinecosystem.move-kin-android:appsDiscovery::<latest release>'
+}
+```
+
+For the latest release version go to [https://github.com/kinecosystem/move-kin-android/releases](https://github.com/kinecosystem/move-kin-android/releases).
+
+
+
+## Explore the Ecosystem
+
+- Adding a button that starts the experience of exploring the Ecosystem - button *AppsDiscoveryButton* 
+
+You can embed the button in any of your layout xml files
+click on the button starts the exploring experience 
+
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout>
+    ...
+    <org.kinecosystem.appsdiscovery.sender.discovery.view.customView.AppsDiscoveryButton
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            app:layout_constraintStart_toStartOf="parent" 
+            app:layout_constraintTop_toTopOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintBottom_toBottomOf="parent"/>
+
+</android.support.constraint.ConstraintLayout>
+```
+
+- Showing explore dialog to the user - *AppsDiscoveryAlertDialog* 
+
+As a trigger to some of a user's actions, you can create and display a dialog *AppsDiscoveryAlertDialog* to the user  
+The user can then choose to start exploring the Ecosystem
+
+```java
+    AppsDiscoveryAlertDialog dialog = new AppsDiscoveryAlertDialog(context);
+    dialog.show();
+```
+
+You can create any button that follows your application style and on a click of a button, it starts the whole experience
+by starting an activity *AppsDiscoveryActivity* 
+
+```java
+someButton.setOnClickListener(v -> {
+            Intent intent = AppsDiscoveryActivity.Companion.getIntent(context)
+             startActivity(intent);
+        });
+```
+
+## Design & UX
+
+For design & UX recommendations and tips on the way to give your users the best experience please follow  [https://discover.kin.org/ux_guidlines_v1.pdf](https://discover.kin.org/ux_guidlines_v1.pdf).
+
+
+## Send Kin
+In order for your app to be able to send Kin
+
+- In your root project, add a new package named *kindiscover*
+- In kindiscover directory - create a service class named *SendKinService*, declare it in a *manifest.xml* file
+and configure the service as not to be exported
+
+```xml
+<application>
+    ...
+    <service android:name=".kindiscover.SendKinService"
+             android:exported="false" />
+</application>
+```
+
+**Notice!** it is important to define the service as **not to be exported**
```

Comment:
You just said that above. Don't think you need to say it again. You could emphasise it / provide a reason when you first mention it.